app/2.0/docs/devguide/properties.md

```diff
@@ -376,7 +376,7 @@ In specific cases, it may be useful to keep an HTML attribute value in sync with
 a property value.  This may be achieved by setting `reflectToAttribute: true` on
 a property in the `properties` configuration object.  This will cause any
 [observable change](data-system#observable-changes) to the property to be serialized out to an
-attribute of the same name.
+attribute according to the reverse mapping as described in <a href="#property-name-mapping">Property name to attribute name mapping</a>.
```

Comment:
Going to defer to @katejeffreys on this because I've read this stuff so many times...

Does this make sense? 

Or, "serialized out to the corresponding attribute as described in ..."

Or (because we don't define serialized until the next section), perhaps should be something like: 

This causes any observable change to the property to trigger an update to the corresponding attribute (as described in Property name to attribute name mapping). Since attributes only take string values, the property value is serialized to a string, as described in Attribute serialization(link).

Reply:
I like this best:

This causes any observable change to the property to trigger an update to the corresponding attribute (as described in Property name to attribute name mapping). Since attributes only take string values, the property value is serialized to a string, as described in Attribute serialization(link).